FlorenceBML/BMLsmarco741/BMLsmarco741.xml

```diff
@@ -35,6 +36,28 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                             <idno>Marrassini ms. 8</idno>
                         </altIdentifier>
                     </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <locus from="6r" to="32v"/>
+                            <title>Śərʿāta qəddāse</title>
+                            <incipit xml:lang="gez">በስመ፡ አብ፡ ወወልድ፡ ወመንፈሰ፡ ቅዱስ፡ አሐዱ፡ እምላክ፡ ዝንቱ፡ ሥርዐተ፡ ቅዳሴ፡ ዘይደሉ፡ ከመ፡ ይበሉ፡ ቀሲስ፡ ወዲያቆን፡ ወሕዝብ፡ 
+                                ኵሎ፡ ዘመፍትው፡ በበጊዜሁ፡ በከመ፡ ሥርዓተ፡ አበዊነ፡ ግብጻዊያን፡</incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <locus from="33r" to="66r"/>
+                            <title ref="LIT1098Anapho"/>
```

Comment:
Do you know if the works are contained in the manuscript completely?

Reply:
Good, then according to your judgement (I also think that the catalogue would have mentioned incompleteness), you could add type="complete" to the title elements.